common/src/main/java/io/seata/common/loader/EnhancedServiceLoader.java

```diff
@@ -516,6 +520,18 @@ private ExtensionDefinition getUnloadedExtensionDefinition(Class<?> clazz) {
             return null;
         }
 
+        private boolean isDefinitionContainsClazz(String className, ClassLoader loader) {
+            for (Map.Entry<Class<?>, ExtensionDefinition> entry : classToDefinitionMap.entrySet()) {
+                if (!entry.getKey().getName().equals(className)) {
+                    continue;
+                }
+                if (entry.getValue().getServiceClass().getClassLoader().equals(loader)) {
```

Comment:
the loader may be null? use Objects.equals ?

Reply:
classLoader alway not null